src/features/FeatureExtractor.js

```diff
@@ -278,10 +280,20 @@ class FeatureExtractor {
      * @param {*} requestSize
      */
     _handleSDPRequest = (dumpLineObj, requestSize) => {
+        const [ , , sdpData ] = dumpLineObj;
         const { metrics } = this.features;
 
-        metrics.sdpRequestBytes += requestSize;
         metrics.sdpRequestCount++;
+
+        if (sdpData.sdpCompressed) {
+            metrics.compressedSdpBytes += requestSize;
+
+            const sdp = lzstring.decompress(sdpData.sdpCompressed);
```

Comment:
I think doing a costly operation such as decompressing just for the sake of knowing the actual side of the sdp might be a bit of an overkill, we're probably fine with just the size of the sdp be it compress or not, considering that that's what will actually consume bandwidth.